amun/base/argo-workflows/inspection-build-with-cpu.yaml

```diff
@@ -267,10 +267,18 @@ spec:
           mkdir -p "$results"
           kubectl logs "{{inputs.parameters.inspection-id}}-1-build" > "${results}/log"
 
-          cat <<< "{{inputs.parameters.specification}}" | jq . > "${results}/specification"
+          spec=`cat <<__EOF__
```

Comment:
can we not use direct echo ?
```suggestion
       echo "{{inputs.parameters.specification}}" | jq . > "${results}/specification"
```

Reply:
We cannot. `{{inputs.parameters.specification}}` part is simply replaced with the parameter value and this causes issues with matching " (it holds a JSON).

Reply:
ok thanks, lets get this in.